sos/plugins/foreman.py

```diff
@@ -17,13 +19,136 @@ class Foreman(Plugin, RedHatPlugin):
 
     plugin_name = 'foreman'
     profiles = ('sysmgmt',)
-    packages = ('foreman-debug',)
+    packages = ('foreman', 'foreman-proxy')
+    option_list = [
+        ('months', 'number of months for dynflow output', 'fast', 6)
+    ]
 
     def setup(self):
-        cmd = "foreman-debug"
+        self.add_forbidden_path([
+            "/etc/foreman*/*key.pem",
+            "/etc/foreman*/encryption_key.rb"
+        ])
 
-        path = self.get_cmd_output_path(name="foreman-debug")
-        self.add_cmd_output("%s -g -q -a -d %s" % (cmd, path),
-                            chroot=self.tmp_in_sysroot(), timeout=900)
+        self.add_copy_spec([
+            "/etc/foreman/",
+            "/etc/foreman-proxy/",
+            "/etc/sysconfig/foreman",
+            "/etc/default/foreman",
+            "/etc/foreman-installer/",
+            "/var/log/foreman/dynflow_executor*.log*",
+            "/var/log/foreman/dynflow_executor*.output*",
+            "/var/log/foreman/apipie_cache*.log*",
+            "/var/log/foreman/cron*.log*",
+            "/var/log/foreman/db_migrate*.log*",
+            "/var/log/foreman/db_seed*.log*",
+            "/var/log/foreman/production.log*",
```

Comment:
This logfile is a crucial and big one in foreman. Daily log-rotated, we need some history of those arbitrary-big files. Again the dilema "how further of history" vs. "keep reasonable small sosreport archive" for small and huge (foreman) deployments..

Could we collect `/var/log/foreman/production.log` separately, and then `/var/log/foreman/production.log-*` (logrotated versions) with `log_size=100` ? (that is still quite low value, imho, but I am bit hesitant to ask much more - any opinion how much logsize we shall collect here, @lzap? I saw `production.log` of sizes 500kB to (extreme case) 30GB)

Reply:
I am leaving you to decide @pmoravec to me it looks sane to always have `production.log` with higher limit and rotated files with lower limit.